src/lib/bootloader/systeminfo.rb

```diff
@@ -18,7 +18,7 @@ class << self
       #
       # @return [Boolean] true if secure boot is currently active
       def secure_boot_active?
-        (efi_supported? || s390_secure_boot_supported? || ppc_secure_boot_active?) &&
+        (efi_supported? || s390_secure_boot_supported? || ppc_secure_boot_supported?) &&
           Sysconfig.from_system.secure_boot
```

Comment:
I suppose this is the actual fix although the commit message is very misleading

Reply:
well, fix contain two parts first is here that state of checkbox reflect OS settings only. The second is special widget which I will explain at second part.